option.go

```diff
@@ -2,6 +2,7 @@ package router
 
 import (
 	"encoding/json"
+	"fmt"
 	"strings"
 	"time"
 
```

Comment:
Go 1.15 or later have time/tzinfo package. https://golang.org/doc/go1.15#time/tzdata
`import _ "time/tzdata"` enables to embed tzdata into an application binary, this is useful for runtime which have not a tzdata.

Reply:
👍

Reply:
https://github.com/kayac/s3-object-router/pull/7/commits/397048bbc417eb00bdbec031798615c8ff98343f